doc/source/release/1.16.0-notes.rst

```diff
@@ -127,8 +153,18 @@ Array API Standard Support
 Experimental support for array libraries other than NumPy has been added to
 existing sub-packages in recent versions of SciPy. Please consider testing
 these features by setting an environment variable ``SCIPY_ARRAY_API=1`` and
-providing PyTorch, JAX, or CuPy arrays as array arguments.
-Features with support added for SciPy 1.16.0 include:
+providing PyTorch, JAX, or CuPy arrays as array arguments. Many functions
+in `scipy.stats`, `scipy.special`, `scipy.optimize`, and `scipy.constants`
```

Comment:
```suggestion
in `scipy.stats`, `scipy.special`, `scipy.optimize`, `scipy.constants`, and `scipy.cluster`
```
depending if https://github.com/scipy/scipy/pull/22960 gets merged

Reply:
K. If it's not merged before this PR merges, please keep this in mind as something to add to scipy/scipy#22991 directly.